tests/Feature/LoginTest.php

```diff
@@ -15,29 +15,51 @@ class LoginTest extends TestCase
      *
      * @return void
      */
-    public function testLoginAndAuth(): void
+    public function testLoginWithCredentials(): void
     {
 
         $user = User::factory()->create();
         $p = $user->password;
         $user->password = Hash::make($user->password);
         $user->save();
 
-        $response = $this->actingAs($user)->json('POST', '/login', [
+        $response = $this->json('POST', '/login', [
             'email' => $user->email,
-            'password' => $p ]
-        );
-        $response->assertStatus(302);
+            'password' => $p
+        ]);
+
+        $props = $response->getOriginalContent()->getData()['page']['props'];
+
+        $this->assertArrayHasKey('success', $props);
+        $this->assertTrue($props['success'] === 'User credentials valid.');
+        $response->assertStatus(200);
+    }
 
+    public function testLoginWithoutPassword(): void
+    {
+        $user = User::factory()->make();
         $response = $this->json('POST', '/login', [
-            'email' => $user->email,
-            'password' => $user->password ]
-        );
-        $response->assertStatus(401);
+            'email' => $user->email
+        ]);
```

Comment:
not sure when I screwed up these tests but once this PU is merged, i'll fix the organization-related tests and add in some github actions to verify tests on each PR